config/routes.rb

```diff
@@ -296,9 +293,9 @@
 
   post '/products' => 'products#create', as: :products
 
-  get '/products/:id', to: redirect(ProductRedirector.new), as: :full_product
-  get '/products/:product_id/discussions/:id', to: redirect(ProductRedirector.new(:discussion)), as: :full_product_discussion
-  get '/products/:product_id/tasks/:id', to: redirect(ProductRedirector.new(:task)), as: :full_product_task
+  get '/products/:id' => redirect(ProductRedirector.new), as: :full_product
+  get '/products/:product_id/discussions/:id' => redirect(ProductRedirector.new(:discussion)), as: :full_product_discussion
+  get '/products/:product_id/tasks/:id' => redirect(ProductRedirector.new(:task)), as: :full_product_task
```

Comment:
Line is too long. [105/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.